src/applications/vaos/new-appointment/index.jsx

```diff
@@ -99,11 +96,7 @@ export function NewAppointment() {
           <TypeOfAudiologyCarePage />
         </Route>
         <Route path={`${match.url}/preferred-date`}>
-          {featureUseVaDate ? (
-            <PreferredDatePageVaDate />
-          ) : (
-            <PreferredDatePage />
```

Comment:
Given that `PreferredDatePage` is no longer used anywhere, should we delete that component from the repository?

Reply:
Yup, we can delete it now that the breadcrumbs feature toggle have been completely removed. But instead of removing unused files/components one at a time, I think it's time we file a spike ticket to review all of our code for unused components and functions. What do you think?

Reply:
Yes, good idea. Let's do the spike to look for unused components.

Reply:
https://github.com/department-of-veterans-affairs/va.gov-team/issues/109361